domains/winter.json

```diff
@@ -1,9 +1,12 @@
 {
     "owner": {
         "username": "virtualWinter",
-        "email": "me@winter-is-a-dev.ml"
+        "email": "v1rtualWinter@outlook.com",
+        "discord": "@vwinter"
     },
     "record": {
-        "A": ["45.14.224.83"]
+        "A": ["217.119.143.38"],
+        "MX": ["winter.is-a.dev"]
```

Comment:
MX records are just a loop of this domain.

Reply:
mail server on the same ip as webhosting